commentsComponent/database/seed.js

```diff
@@ -0,0 +1,12 @@
+var data = require('./database/seed_data.js/index.js');
+var mysql = require('mysql');
+
+
+
+
+var seedDb = function(data) {
+//function needed to populate db with 
```

Comment:
When you start implementing seedDB...I would recommend writing your mysql queries in a separate file and then importing them via require. Makes your code a lot cleaner and easier to read.